UPGRADE-7.4.md

```diff
@@ -59,6 +59,7 @@ FrameworkBundle
 ---------------
 
  * Deprecate `Symfony\Bundle\FrameworkBundle\Console\Application::add()` in favor of `addCommand()`
+ * Command `Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand` has been moved to `Symfony\Component\Workflow\Command\WorkflowDumpCommand`
```

Comment:
```suggestion
 * Deprecate `Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand` in favor of `Symfony\Component\Workflow\Command\WorkflowDumpCommand`
```

`Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand` still exists for BC reasons.